docs/en/contributor/Documentation-Conventions.md

```diff
@@ -97,7 +97,7 @@ this has been the norm [as of the 1950s](https://en.wikipedia.org/wiki/Sentence_
 |-------------------------|-----------------------------------------------------|
 | File system | Filesystem |
 | Leading master | Leader, lead master, primary master |
-| Backup master | Secondary master, following master, follower master |
+| Backup master | Standby master, following master, follower master |
```

Comment:
let's go with `Standby master` as the preferred term and `Backup master` as less preferred to reflect these changes

Reply:
Thanks. I change it.